### PR TITLE
Group controls editor config

### PR DESCRIPTION
**Pull request by goshander**
/link CHARTS-6728

Plugin `GroupControl` provide as is, without correct render

### CODE REVIEW

**Review by Marginy605 — CHANGES_REQUESTED** [review]

File: src/ui/components/DashKit/plugins/GroupControl/GroupControl.tsx [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+        return Promise.resolve({});
+    }
+
+    get actualParams(): StringParams {
+        return this.props.params;
+    }
+
+    async init() {
+        try {
+            // const {data} = this.props;
```
Comment (mournfulCoroner) [inline src/ui/components/DashKit/plugins/GroupControl/GroupControl.tsx:215]: One more unnecessary comment?

File: src/ui/units/dash/containers/Dialogs/GroupControl/ControlsPlacementDialog/ControlsPlacementDialog.tsx [hunk 2]
```diff
@@ -71,7 +71,10 @@ const ControlsPlacementDialog = ({onClose}: ControlsPlacementDialogProps) => {
             const dragItem = prevItemsState[oldIndex];
             const newItemsState = prevItemsState.filter((_, index) => index !== oldIndex);
             newItemsState.splice(newIndex, 0, dragItem);
-            return newItemsState;
+            return {
+                ...prevItemsState,
+                items: newItemsState,
```
Comment (mournfulCoroner) [inline src/ui/units/dash/containers/Dialogs/GroupControl/ControlsPlacementDialog/ControlsPlacementDialog.tsx:76]: Is there a `items` field definitely present in `itemsState`? The initial value is `selectorsGroup.items`

Comment (goshander) [reply src/ui/units/dash/containers/Dialogs/GroupControl/ControlsPlacementDialog/ControlsPlacementDialog.tsx:76]: Oh, my bad, thank you

**Comment by robot-charts** [comment]
**i18n-check**
<details>
<summary>The following components have not been translated</summary>
<p><a href="https://weblate.gravity-ui.com/projects/datalens/group-controls-editor-config-109/dash-popup-widgets-order-view-109/en/">dash.popup-widgets-order.view (en)</a><br><a href="https://weblate.gravity-ui.com/projects/datalens/group-controls-editor-config-109/dash-popup-widgets-order-view-109/ru/">dash.popup-widgets-order.view (ru)</a><br><a href="https://weblate.gravity-ui.com/projects/datalens/group-controls-editor-config-109/temp-2-109/en/">temp2 (en)</a><br><a href="https://weblate.gravity-ui.com/projects/datalens/group-controls-editor-config-109/temp-2-109/ru/">temp2 (ru)</a><br><a href="https://weblate.gravity-ui.com/projects/datalens/group-controls-editor-config-109/temp-3-109/en/">temp3 (en)</a><br><a href="https://weblate.gravity-ui.com/projects/datalens/group-controls-editor-config-109/temp-3-109/ru/">temp3 (ru)</a></p>
</details>

Wait for the reviewers to check your changes in Weblate and try running github action again.

**Comment by robot-charts** [comment]
**i18n-check**
<details>
<summary>The following components have not been translated</summary>
<p><a href="https://weblate.gravity-ui.com/projects/datalens/group-controls-editor-config-109/dash-popup-widgets-order-view-109/en/">dash.popup-widgets-order.view (en)</a><br><a href="https://weblate.gravity-ui.com/projects/datalens/group-controls-editor-config-109/dash-popup-widgets-order-view-109/ru/">dash.popup-widgets-order.view (ru)</a><br><a href="https://weblate.gravity-ui.com/projects/datalens/group-controls-editor-config-109/sql-109/en/">sql (en)</a><br><a href="https://weblate.gravity-ui.com/projects/datalens/group-controls-editor-config-109/sql-109/ru/">sql (ru)</a></p>
</details>

Wait for the reviewers to check your changes in Weblate and try running github action again.

**Comment by robot-charts** [comment]
**i18n-check**
<details>
<summary>The following components have not been translated</summary>
<p><a href="https://weblate.gravity-ui.com/projects/datalens/group-controls-editor-config-109/dash-popup-widgets-order-view-109/en/">dash.popup-widgets-order.view (en)</a><br><a href="https://weblate.gravity-ui.com/projects/datalens/group-controls-editor-config-109/dash-popup-widgets-order-view-109/ru/">dash.popup-widgets-order.view (ru)</a><br><a href="https://weblate.gravity-ui.com/projects/datalens/group-controls-editor-config-109/sql-109/en/">sql (en)</a><br><a href="https://weblate.gravity-ui.com/projects/datalens/group-controls-editor-config-109/sql-109/ru/">sql (ru)</a></p>
</details>

Wait for the reviewers to check your changes in Weblate and try running github action again.

**Comment by robot-charts** [comment]
**i18n-check**
<details>
<summary>The following components have not been translated</summary>
<p><a href="https://weblate.gravity-ui.com/projects/datalens/group-controls-editor-config-109/dash-popup-widgets-order-view-109/en/">dash.popup-widgets-order.view (en)</a><br><a href="https://weblate.gravity-ui.com/projects/datalens/group-controls-editor-config-109/dash-popup-widgets-order-view-109/ru/">dash.popup-widgets-order.view (ru)</a><br><a href="https://weblate.gravity-ui.com/projects/datalens/group-controls-editor-config-109/sql-109/en/">sql (en)</a><br><a href="https://weblate.gravity-ui.com/projects/datalens/group-controls-editor-config-109/sql-109/ru/">sql (ru)</a></p>
</details>

Wait for the reviewers to check your changes in Weblate and try running github action again.

**Comment by robot-charts** [comment]
**i18n-check**
<details>
<summary>The following components have not been translated</summary>
<p><a href="https://weblate.gravity-ui.com/projects/datalens/group-controls-editor-config-109/dash-popup-widgets-order-view-109/en/">dash.popup-widgets-order.view (en)</a><br><a href="https://weblate.gravity-ui.com/projects/datalens/group-controls-editor-config-109/dash-popup-widgets-order-view-109/ru/">dash.popup-widgets-order.view (ru)</a><br><a href="https://weblate.gravity-ui.com/projects/datalens/group-controls-editor-config-109/sql-109/en/">sql (en)</a><br><a href="https://weblate.gravity-ui.com/projects/datalens/group-controls-editor-config-109/sql-109/ru/">sql (ru)</a></p>
</details>

Wait for the reviewers to check your changes in Weblate and try running github action again.

**Comment by robot-charts** [comment]
**i18n-check**
<details>
<summary>Errors occurred when merging changes from your branch with the Weblate branch.</summary>
```
warning: skipped previously applied commit de351d4d
hint: use --reapply-cherry-picks to include skipped commits
hint: Disable this message with "git config advice.skippedCherryPicks false"
Rebasing (1/1)
Auto-merging .lintstagedrc.json
CONFLICT (add/add): Merge conflict in .lintstagedrc.json
Auto-merging README.md
CONFLICT (add/add): Merge conflict in README.md
Auto-merging package-lock.json
CONFLICT (add/add): Merge conflict in package-lock.json
Auto-merging package.json
CONFLICT (add/add): Merge conflict in package.json
Auto-merging src/i18n-keysets/collections/ru.json
CONFLICT (add/add): Merge conflict in src/i18n-keysets/collections/ru.json
Auto-merging src/i18n-keysets/sql/en.json
CONFLICT (add/add): Merge conflict in src/i18n-keysets/sql/en.json
Auto-merging src/i18n-keysets/sql/keyset.json
CONFLICT (add/add): Merge conflict in src/i18n-keysets/sql/keyset.json
Auto-merging src/i18n-keysets/sql/ru.json
CONFLICT (add/add): Merge conflict in src/i18n-keysets/sql/ru.json
Auto-merging src/server/components/features/features-list/GroupControls.ts
CONFLICT (add/add): Merge conflict in src/server/components/features/features-list/GroupControls.ts
Auto-merging src/server/modes/charts/plugins/datalens/preparers/line/d3.ts
CONFLICT (add/add): Merge conflict in src/server/modes/charts/plugins/datalens/preparers/line/d3.ts
Auto-merging src/server/modes/charts/plugins/datalens/preparers/line/prepare-line-data.ts
CONFLICT (add/add): Merge conflict in src/server/modes/charts/plugins/datalens/preparers/line/prepare-line-data.ts
Auto-merging src/server/modes/charts/plugins/datalens/preparers/scatter/d3.ts
CONFLICT (add/add): Merge conflict in src/server/modes/charts/plugins/datalens/preparers/scatter/d3.ts
Auto-merging src/shared/constants/qa/components.ts
CONFLICT (add/add): Merge conflict in src/shared/constants/qa/components.ts
Auto-merging src/shared/types/feature.ts
CONFLICT (add/add): Merge conflict in src/shared/types/feature.ts
Auto-merging src/ui/components/DashKit/DashKit.ts
CONFLICT (add/add): Merge conflict in src/ui/components/DashKit/DashKit.ts
Auto-merging src/ui/components/DashKit/utils.ts
CONFLICT (add/add): Merge conflict in src/ui/components/DashKit/utils.ts
Auto-merging src/ui/components/Widgets/Chart/ChartSelector.tsx
CONFLICT (add/add): Merge conflict in src/ui/components/Widgets/Chart/ChartSelector.tsx
Auto-merging src/ui/components/Widgets/Chart/ChartWidget.scss
CONFLICT (add/add): Merge conflict in src/ui/components/Widgets/Chart/ChartWidget.scss
Auto-merging src/ui/components/Widgets/Chart/ChartWidget.tsx
CONFLICT (add/add): Merge conflict in src/ui/components/Widgets/Chart/ChartWidget.tsx
Auto-merging src/ui/components/Widgets/Chart/hooks/useLoadingChart.ts
CONFLICT (add/add): Merge conflict in src/ui/components/Widgets/Chart/hooks/useLoadingChart.ts
Auto-merging src/ui/components/Widgets/Chart/hooks/useLoadingChartSelector.tsx
CONFLICT (add/add): Merge conflict in src/ui/components/Widgets/Chart/hooks/useLoadingChartSelector.tsx
Auto-merging src/ui/components/Widgets/Chart/hooks/useLoadingChartWidget.ts
CONFLICT (add/add): Merge conflict in src/ui/components/Widgets/Chart/hooks/useLoadingChartWidget.ts
Auto-merging src/ui/constants/visualizations/line.ts
CONFLICT (add/add): Merge conflict in src/ui/constants/visualizations/line.ts
Auto-merging src/ui/constants/visualizations/scatter.ts
CONFLICT (add/add): Merge conflict in src/ui/constants/visualizations/scatter.ts
Auto-merging src/ui/libs/DatalensChartkit/ChartKit/components/Widget/components/Table/Table.tsx
CONFLICT (add/add): Merge conflict in src/ui/libs/DatalensChartkit/ChartKit/components/Widget/components/Table/Table.tsx
Auto-merging src/ui/libs/DatalensChartkit/ChartKit/components/Widget/components/Table/__tests__/misc.test.ts
CONFLICT (add/add): Merge conflict in src/ui/libs/DatalensChartkit/ChartKit/components/Widget/components/Table/__tests__/misc.test.ts
Auto-merging src/ui/libs/DatalensChartkit/ChartKit/components/Widget/components/Table/utils/event-handlers.ts
CONFLICT (add/add): Merge conflict in src/ui/libs/DatalensChartkit/ChartKit/components/Widget/components/Table/utils/event-handlers.ts
Auto-merging src/ui/libs/DatalensChartkit/ChartKit/components/Widget/components/Table/utils/misc.ts
CONFLICT (add/add): Merge conflict in src/ui/libs/DatalensChartkit/ChartKit/components/Widget/components/Table/utils/misc.ts
Auto-merging src/ui/libs/DatalensChartkit/ChartKit/components/Widget/components/Table/utils/render.tsx
CONFLICT (add/add): Merge conflict in src/ui/libs/DatalensChartkit/ChartKit/components/Widget/components/Table/utils/render.tsx
Auto-merging src/ui/libs/DatalensChartkit/ChartKit/helpers/action-params-handlers.ts
CONFLICT (add/add): Merge conflict in src/ui/libs/DatalensChartkit/ChartKit/helpers/action-params-handlers.ts
Auto-merging src/ui/libs/DatalensChartkit/ChartKit/helpers/apply-hc-handlers.ts
CONFLICT (add/add): Merge conflict in src/ui/libs/DatalensChartkit/ChartKit/helpers/apply-hc-handlers.ts
Auto-merging src/ui/units/connections/components/custom-forms/Yadocs/components/AdditionalTitleContent.tsx
CONFLICT (add/add): Merge conflict in src/ui/units/connections/components/custom-forms/Yadocs/components/AdditionalTitleContent.tsx
Auto-merging src/ui/units/connections/components/custom-forms/Yadocs/components/DialogAddDocument/DialogAddDocument.tsx
CONFLICT (add/add): Merge conflict in src/ui/units/connections/components/custom-forms/Yadocs/components/DialogAddDocument/DialogAddDocument.tsx
Auto-merging src/ui/units/connections/components/custom-forms/Yadocs/components/DocsList.tsx
CONFLICT (add/add): Merge conflict in src/ui/units/connections/components/custom-forms/Yadocs/components/DocsList.tsx
Auto-merging src/ui/units/connections/components/custom-forms/Yadocs/components/Workspace.tsx
CONFLICT (add/add): Merge conflict in src/ui/units/connections/components/custom-forms/Yadocs/components/Workspace.tsx
Auto-merging src/ui/units/connections/components/custom-forms/Yadocs/containers/ActionBarContainer.tsx
CONFLICT (add/add): Merge conflict in src/ui/units/connections/components/custom-forms/Yadocs/containers/ActionBarContainer.tsx
Auto-merging src/ui/units/connections/components/custom-forms/Yadocs/containers/useYadocsDialogs.ts
CONFLICT (add/add): Merge conflict in src/ui/units/connections/components/custom-forms/Yadocs/containers/useYadocsDialogs.ts
Auto-merging src/ui/units/connections/utils/yadocs.ts
CONFLICT (add/add): Merge conflict in src/ui/units/connections/utils/yadocs.ts
Auto-merging src/ui/units/dash/components/DashActionPanel/EditControls/EditControls.tsx
CONFLICT (add/add): Merge conflict in src/ui/units/dash/components/DashActionPanel/EditControls/EditControls.tsx
Auto-merging src/ui/units/dash/containers/Dialogs/DialogRelations/DialogRelations.tsx
CONFLICT (add/add): Merge conflict in src/ui/units/dash/containers/Dialogs/DialogRelations/DialogRelations.tsx
Auto-merging src/ui/units/dash/containers/Dialogs/Settings/Settings.tsx
CONFLICT (add/add): Merge conflict in src/ui/units/dash/containers/Dialogs/Settings/Settings.tsx
Auto-merging src/ui/units/dash/containers/Dialogs/Tabs/PopupWidgetsOrder/helpers.ts
CONFLICT (add/add): Merge conflict in src/ui/units/dash/containers/Dialogs/Tabs/PopupWidgetsOrder/helpers.ts
Auto-merging src/ui/units/dash/store/actions/dashTyped.ts
CONFLICT (add/add): Merge conflict in src/ui/units/dash/store/actions/dashTyped.ts
Auto-merging src/ui/units/dash/store/reducers/dash.js
CONFLICT (add/add): Merge conflict in src/ui/units/dash/store/reducers/dash.js
Auto-merging src/ui/units/ql/constants/index.ts
CONFLICT (add/add): Merge conflict in src/ui/units/ql/constants/index.ts
Auto-merging src/ui/units/ql/containers/PaneMain/ScreenEditor/ScreenEditor.scss
CONFLICT (add/add): Merge conflict in src/ui/units/ql/containers/PaneMain/ScreenEditor/ScreenEditor.scss
Auto-merging src/ui/units/ql/containers/PaneMain/ScreenEditor/ScreenEditor.tsx
CONFLICT (add/add): Merge conflict in src/ui/units/ql/containers/PaneMain/ScreenEditor/ScreenEditor.tsx
Auto-merging src/ui/units/ql/containers/PaneVisualization/PaneVisualization.tsx
CONFLICT (add/add): Merge conflict in src/ui/units/ql/containers/PaneVisualization/PaneVisualization.tsx
Auto-merging src/ui/units/ql/store/actions/ql.ts
CONFLICT (add/add): Merge conflict in src/ui/units/ql/store/actions/ql.ts
Auto-merging src/ui/units/ql/store/reducers/ql.ts
CONFLICT (add/add): Merge conflict in src/ui/units/ql/store/reducers/ql.ts
Auto-merging src/ui/units/ql/store/typings/ql.ts
CONFLICT (add/add): Merge conflict in src/ui/units/ql/store/typings/ql.ts
Auto-merging src/ui/units/wizard/actions/dialog.ts
CONFLICT (add/add): Merge conflict in src/ui/units/wizard/actions/dialog.ts
Auto-merging src/ui/units/wizard/containers/Wizard/SectionVisualization/PlaceholdersContainer/ShapesPlaceholder/ShapesPlaceholder.tsx
CONFLICT (add/add): Merge conflict in src/ui/units/wizard/containers/Wizard/SectionVisualization/PlaceholdersContainer/ShapesPlaceholder/ShapesPlaceholder.tsx
Auto-merging tests/opensource-suites/dash/base/relations.test.ts
CONFLICT (add/add): Merge conflict in tests/opensource-suites/dash/base/relations.test.ts
Auto-merging tests/opensource-suites/dash/params/params.test.ts
CONFLICT (add/add): Merge conflict in tests/opensource-suites/dash/params/params.test.ts
Auto-merging tests/page-objects/dashboard/DashboardPage.ts
CONFLICT (add/add): Merge conflict in tests/page-objects/dashboard/DashboardPage.ts
Auto-merging tests/suites/dash/base/actionPanel.test.ts
CONFLICT (add/add): Merge conflict in tests/suites/dash/base/actionPanel.test.ts
Auto-merging tests/suites/dash/base/relations.test.ts
CONFLICT (add/add): Merge conflict in tests/suites/dash/base/relations.test.ts
Auto-merging tests/suites/dash/mobile/mobileDash.test.ts
CONFLICT (add/add): Merge conflict in tests/suites/dash/mobile/mobileDash.test.ts
Auto-merging tests/suites/dash/params/params.test.ts
CONFLICT (add/add): Merge conflict in tests/suites/dash/params/params.test.ts
Auto-merging tests/suites/dash/revisions/revisionsListWithCreationDashboards.test.ts
CONFLICT (add/add): Merge conflict in tests/suites/dash/revisions/revisionsListWithCreationDashboards.test.ts
Auto-merging tests/suites/dash/revisions/revisionsListWithCreationSaveAsCopy.test.ts
CONFLICT (add/add): Merge conflict in tests/suites/dash/revisions/revisionsListWithCreationSaveAsCopy.test.ts
Auto-merging tests/suites/dash/selectors/addSelectors.test.ts
CONFLICT (add/add): Merge conflict in tests/suites/dash/selectors/addSelectors.test.ts
Auto-merging tests/suites/dash/widgets/autoheight.test.ts
CONFLICT (add/add): Merge conflict in tests/suites/dash/widgets/autoheight.test.ts
error: could not apply 50be1ba6... fix widget order
hint: Resolve all conflicts manually, mark them as resolved with
hint: "git add/rm <conflicted_files>", then run "git rebase --continue".
hint: You can instead skip this commit: run "git rebase --skip".
hint: To abort and get back to the state before "git rebase", run "git rebase --abort".
Could not apply 50be1ba6... fix widget order
 (1)
```
</details>
**Resolve conflicts according to instructions**
1. Switch to the current branch associated with this pull request.
```
git checkout group-controls-editor-config
```
2. Add Weblate as remote:
```
git remote add weblate https://weblate.gravity-ui.com/git/datalens/group-controls-editor-config-109/chartkit-109/
git remote update weblate
```
3. Merge Weblate changes:
```
git merge weblate/group-controls-editor-config
```
4. Resolve conflicts:
```
edit ...
git add ...
git commit
```
5. Push changes to upstream repository, Weblate will fetch merge from there:
```
git push origin
```

**Comment by robot-charts** [comment]
**i18n-check**
<details>
<summary>The following components have not been translated</summary>
<p><a href="https://weblate.gravity-ui.com/projects/datalens/group-controls-editor-config-109/dash-popup-widgets-order-view-109/en/">dash.popup-widgets-order.view (en)</a><br><a href="https://weblate.gravity-ui.com/projects/datalens/group-controls-editor-config-109/dash-popup-widgets-order-view-109/ru/">dash.popup-widgets-order.view (ru)</a><br><a href="https://weblate.gravity-ui.com/projects/datalens/group-controls-editor-config-109/sql-109/en/">sql (en)</a><br><a href="https://weblate.gravity-ui.com/projects/datalens/group-controls-editor-config-109/sql-109/ru/">sql (ru)</a></p>
</details>

Wait for the reviewers to check your changes in Weblate and try running github action again.

**Comment by robot-charts** [comment]
**i18n-check**
<details>
<summary>The following components have not been translated</summary>
<p><a href="https://weblate.gravity-ui.com/projects/datalens/group-controls-editor-config-109/dash-popup-widgets-order-view-109/en/">dash.popup-widgets-order.view (en)</a><br><a href="https://weblate.gravity-ui.com/projects/datalens/group-controls-editor-config-109/dash-popup-widgets-order-view-109/ru/">dash.popup-widgets-order.view (ru)</a><br><a href="https://weblate.gravity-ui.com/projects/datalens/group-controls-editor-config-109/sql-109/en/">sql (en)</a><br><a href="https://weblate.gravity-ui.com/projects/datalens/group-controls-editor-config-109/sql-109/ru/">sql (ru)</a></p>
</details>

Wait for the reviewers to check your changes in Weblate and try running github action again.

**Comment by robot-charts** [comment]
**i18n-check**
<details>
<summary>The following components have not been translated</summary>
<p><a href="https://weblate.gravity-ui.com/projects/datalens/group-controls-editor-config-109/dash-popup-widgets-order-view-109/en/">dash.popup-widgets-order.view (en)</a><br><a href="https://weblate.gravity-ui.com/projects/datalens/group-controls-editor-config-109/dash-popup-widgets-order-view-109/ru/">dash.popup-widgets-order.view (ru)</a><br><a href="https://weblate.gravity-ui.com/projects/datalens/group-controls-editor-config-109/sql-109/en/">sql (en)</a><br><a href="https://weblate.gravity-ui.com/projects/datalens/group-controls-editor-config-109/sql-109/ru/">sql (ru)</a></p>
</details>

Wait for the reviewers to check your changes in Weblate and try running github action again.

**Comment by robot-charts** [comment]
**i18n-check**
<details>
<summary>The following components have not been translated</summary>
<p><a href="https://weblate.gravity-ui.com/projects/datalens/group-controls-editor-config-109/dash-popup-widgets-order-view-109/en/">dash.popup-widgets-order.view (en)</a><br><a href="https://weblate.gravity-ui.com/projects/datalens/group-controls-editor-config-109/dash-popup-widgets-order-view-109/ru/">dash.popup-widgets-order.view (ru)</a><br><a href="https://weblate.gravity-ui.com/projects/datalens/group-controls-editor-config-109/sql-109/en/">sql (en)</a><br><a href="https://weblate.gravity-ui.com/projects/datalens/group-controls-editor-config-109/sql-109/ru/">sql (ru)</a></p>
</details>

Wait for the reviewers to check your changes in Weblate and try running github action again.

**Review by mournfulCoroner — APPROVED** [review]

**Comment by robot-charts** [comment]
**i18n-check**
<details>
<summary>The following components have not been translated</summary>
<p><a href="https://weblate.gravity-ui.com/projects/datalens/group-controls-editor-config-109/sql-109/en/">sql (en)</a><br><a href="https://weblate.gravity-ui.com/projects/datalens/group-controls-editor-config-109/sql-109/ru/">sql (ru)</a></p>
</details>

Wait for the reviewers to check your changes in Weblate and try running github action again.

**Review by Marginy605 — APPROVED** [review]

**Pull request merged**